pkg/ovirt/configFuncs.es6

```diff
@@ -37,7 +37,15 @@ export function readConfiguration ({ dispatch }) {
     promises.push(doReadHostname({ dispatch }));
     promises.push(doReadIpAddresses({ dispatch }));
 
-    return cockpit.all(promises);
+    return cockpit.all(promises).done(function () {
+        // If hostname was found by doReadHostname() and VIRSH_CONNECTION_URI wasn't set in CONFIG_FILE
+        // (or CONFIG_FILE wasn't created) then auto calculate the Virsh connection URI as:
+        // qemu+tcp://${hostName}/system
+        if (CONFIG.hostName && MACHINES_CONFIG.Virsh && MACHINES_CONFIG.Virsh.connections &&
+                Object.getOwnPropertyNames(MACHINES_CONFIG.Virsh.connections).indexOf('remote') === -1) {
+            MACHINES_CONFIG.Virsh.connections = {'remote': { params: ['-c', `qemu+tcp://${CONFIG.hostName}/system`] }};
```

Comment:
is qemu+tcp working? it should be tls...

Reply:
qemu+tls is the one and only access you get on oVirt hosts. If anyone mentiones qemu+tcp it's only for devel setup without TLS and that's simply not a supported deployment. For devel use cases one can use the config file to define something else - the default for users should be qemu+tls